src/components/ImportRecordsFlow/stepTwo/__test__/uploadFileButton.spec.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import UploadFileButton from '../uploadFileButton';
+import { uniqueId } from '../../../../libs/utils';
+
+jest.mock('../../../../libs/utils/uniqueId', () => jest.fn(() => '101'));
+
+describe('<UploadFileButton />', () => {
+    it('should have unique id set properly', () => {
+        const component = mount(<UploadFileButton onChange={() => {}} />);
+        const input = component.find('input');
+        expect(uniqueId).toHaveBeenCalledWith('upload-file-input');
+        expect(input.prop('id')).toBe('101');
```

Comment:
I think these should be two tests:
- should call uniqueId with the right value
- should associate the input and label with the same id
in order to check accessibility to this last test case, we need to add another expect that check the value of the htmlFor attribute in the label element